adders/drizzle/config/adder.ts

```diff
@@ -222,7 +224,7 @@ export const adder = defineAdderConfig({
                         integer: "integer",
                     });
 
-                    userSchemaExpression = common.expressionFromString(`sqliteTable('user', {
+                    userSchemaExpression = common.expressionFromString(dedent`sqliteTable('user', {
```

Comment:
could we move the dedent to occur inside `expressionFromString`? or is there ever a time we wouldn't want to do that?

Reply:
heh, this is actually one of the changes that I have uncommitted and pending for #467 and it broke everything for unrelated reasons 😆 I can definitely move it here